src/devices/shinasystem.ts

```diff
@@ -640,15 +661,89 @@ export const definitions: DefinitionWithExtend[] = [
         vendor: "ShinaSystem",
         ota: true,
         description: "SiHAS energy monitor",
-        extend: [m.electricityMeter({power: {cluster: "metering"}, acFrequency: {multiplier: 1, divisor: 10}, powerFactor: true}), m.temperature()],
+        fromZigbee: [fzLocal.ct_direction],
+        toZigbee: [tzLocal.ct_direction],
+        extend: [
+            m.electricityMeter({
+                power: {cluster: "metering", min: 30},
+                current: {min: 30, max: 600, change: 1},
```

Comment:
Let's keep the default min/max/change reporting config here, this gives a consistent experience across devices. In case people want to change it they can change it via the reporting tab.

Reply:
I understand and agree.